self-tests.yml

```diff
@@ -612,6 +612,7 @@ tasks:
           RUN_TEST: "TestEnvironmentSuite/TestInitDB"
           MONGO_CREDS_FILE: testdata/test_auth.yml
   - name: generate-api-docs
+    patchable: false
```

Comment:
If this doesn't run for patches anymore, can we have a separate task for swaggo format linting checks in PRs?

Reply:
Oh shoot, sorry, that was my original intent. I also added an execution number to the non-latest part, so you can see the file in a patch build if you want. I also added it to the PR definitions.